gcc/rust/ast/clone-test.h

```diff
@@ -1,3 +1,21 @@
+// Copyright (C) 2020 Free Software Foundation, Inc.
+
+// This file is part of GCC.
+
+// GCC is free software; you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the Free
+// Software Foundation; either version 3, or (at your option) any later
+// version.
+
+// GCC is distributed in the hope that it will be useful, but WITHOUT ANY
+// WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+// for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with GCC; see the file COPYING3.  If not see
+// <http://www.gnu.org/licenses/>.
+
 #ifndef CLONE_TEST_H
```

Comment:
This file should probably be removed. It isn't actually used in the current codebase for anything - it was a failed test in seeing whether I could get clone to work automatically without the boilerplate (of having to manually define copy constructors and whatever) that currently exists.